src/genie/libs/parser/iosxe/tests/ShowVrrp/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,41 @@
+expected_output = { 
+     {
+    "interface": {
+        "GigabitEthernet1": {
+            "group_number": {
+                "Group 100": {
```

Comment:
simply can have as `group` and value is integer if the value is always number.
```suggestion
            "group": {
                100: {
```

Reply:
confirmed the change.